src/Database/LMDB/Simple/Internal.hs

```diff
@@ -169,9 +173,16 @@ instance MonadIO (Transaction mode) where
 -- "Codec.Serialise.Tutorial".
 data Database k v = Db MDB_env MDB_dbi'
 
+-- | FIXME(jdral): this function should be renamed to @peekMDBVal@ to avoid
```

Comment:
Is this something we can fix now? If not, is it critical? If it is not critical maybe rename `FIXME` to `TODO`.

Reply:
It is not critical, so it should instead be a TODO. It is something we can fix, but it will probably require renaming in a bunch of other places in other modules, so I would prefer to make the change in a separate PR. I'll create an issue for it: https://github.com/input-output-hk/ouroboros-network/issues/4179.